diff a/pkg/sysctl/Dockerfile b/pkg/sysctl/Dockerfile
--- a/pkg/sysctl/Dockerfile
+++ b/pkg/sysctl/Dockerfile
@@ -2,8 +2,9 @@ FROM linuxkit/alpine:33063834cf72d563cd8703467836aaa2f2b5a300 AS mirror
 
 RUN apk add --no-cache go musl-dev
 ENV GOPATH=/go PATH=$PATH:/go/bin GO111MODULE=off
-# Hack to work around an issue with go on arm64 requiring gcc
-RUN [ $(uname -m) = aarch64 ] && apk add --no-cache gcc || true
+
+# Hack to work around an issue with go on arm requiring gcc
+RUN if [ $(uname -m) = aarch64 ] || [ $(uname -m) = armv7l ]; then apk add --no-cache gcc ; fi
 
 COPY main.go /go/src/sysctl/
 RUN go-compile.sh /go/src/sysctl
